gpt_researcher/retrievers/google/google.py

```
@@ -31,7 +31,7 @@ def get_api_key(self):
         # Get the API key
         try:
             api_key = os.environ["GOOGLE_API_KEY"]
-        except:
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)